onnxruntime/python/tools/kernel_explorer/kernels/gemm_test.py

```diff
@@ -29,14 +29,18 @@ def _test_gemm(func, dtype: str, m: int, n: int, k: int, transa=False, transb=Fa
     np.random.seed(0)
     a = (np.random.rand(*a_shape) + 0.5).astype(dtype).astype("float64")
     b = (np.random.rand(*b_shape) + 0.5).astype(dtype).astype("float64")
-    ref_c = (a.T if transa else a) @ (b.T if transb else b)
+    intermidiate_c = (a.T if transa else a) @ (b.T if transb else b)
+    if alpha == 1.0 and beta == 0.0:  # fast path
+        ref_c = intermidiate_c
+    else:
+        ref_c = alpha * intermidiate_c + beta * np.ones_like(intermidiate_c)
 
     bound = get_gemm_bound(dtype, a, b, ref_c, transa, transb)
 
     a = a.astype(dtype)
     b = b.astype(dtype)
 
-    my_c = np.zeros((m, n), dtype=dtype)
+    my_c = np.ones((m, n), dtype=dtype)
```

Comment:
Can you please elaborate why this change is needed?

Reply:
To test beta